src/docs/for-developers/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Getting started for developers
+author: "Santiment team"
+description: Technical documentation on Santiment metrics and indicators. Understand the calculations, logic and algorithms behind our metrics - many of them custom-built by the Santiment team.
+---
+
+## Overview
+
+Santiment API uses [GraphQL](https://graphql.org). From the beginning we decided
+to use GraphQL instead of REST for a number of reasons:
+
+- You can request exactly the data you need and also easily batch requests together.
+  This is effectively handling the issues with underfetching and overfetching data. Why
+  fetching all 20+ fields of a project when you only need its name?
+- The request describes the format of the response. You no longer need
+  to wonder what data the result contains and how to parse it.
+- Easy out-of-the-box way to explore our API via our Live Explorer
+
+## Let's start
+For example we want to get development activity of Ethereum team.
+Copy and paste this curl request to your console:
+
+```bash
+curl -X POST -H "Content-Type: application/graphql" --data '{devActivity(from: "2019-01-01T00:00:00Z", interval: "1d", slug: "ethereum", to: "2019-01-07T00:00:00Z") { activity datetime }}' https://api.santiment.net/graphql
+```
+
+You will see this response:
+
+```bash
+{"data":{"devActivity":[{"activity":0,"datetime":"2019-01-01T00:00:00Z"},{"activity":6,"datetime":"2019-01-02T00:00:00Z"},{"activity":0,"datetime":"2019-01-03T00:00:00Z"},{"activity":4,"datetime":"2019-01-04T00:00:00Z"},{"activity":0,"datetime":"2019-01-05T00:00:00Z"},{"activity":0,"datetime":"2019-01-06T00:00:00Z"}]}}
+```
+
+To get the full list of available metrics you can visit [metrics](https://academy.santiment.net/metrics/)
+For example, for dev activity you can open [this
+page](https://academy.santiment.net/metrics/developer-activity/) to get the full instructions
+
+Also we have a python wrapper for our graphql API, usage of this library is even
+easier.
+
+[Sanpy library](https://github.com/santiment/sanpy)
+
+```
+pip install sanpy
+```
+
+And you can try to get the same ethereum team dev activity data with this python library
+
+```python
+report = san.get(
+    "dev_activity/ethereum",
+    from_date="2019-01-01",
+    to_date="2019-01-07",
+    interval="1d"
+)
+
+print(report)
+```
+
+Example result:
+
+```
+                           activity
+datetime
```

Comment:
both datetime and activity should be on the same row

Reply:
It's the same as real answer. It's not on the same.